test/reduce_test.cpp

```diff
@@ -810,6 +881,9 @@ struct reduce_driver : test_driver
                 this->tolerance = 80 * 10;
         };
 
+        if(std::is_same<T, half_float::half>::value)
+            this->tolerance *= this->tolerance * 10.0;
```

Comment:
This would lead to the constant growth of `tolerance` when called multiple times from `test_drive_impl_1()` via `base_run()`, isn't it?
https://github.com/ROCmSoftwarePlatform/MIOpen/blob/7c34cdeca01e6e4c33b2fb72121aa81a7443fc4e/test/driver.hpp#L1239

Reply:
> This would lead to the constant growth of `tolerance` when called multiple times from `test_drive_impl_1()` via `base_run()`, isn't it?
> https://github.com/ROCmSoftwarePlatform/MIOpen/blob/7c34cdeca01e6e4c33b2fb72121aa81a7443fc4e/test/driver.hpp#L1239

Yes, this needs be corrected.